developer-docs-site/static/examples/python/first_nft.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Aptos
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Any, Dict
+import sys
+
+from first_transaction import Account, FaucetClient, RestClient, TESTNET_URL, FAUCET_URL
+
+class TokenClient(RestClient):
```

Comment:
you really like to write python, add a smoke test in rust too :)

Reply:
Python is good enough for everyone!

Yeah, I'll add a comprehensive test as a smoke test -- once we refine the APIs and have docs.